openhtf/io/proto/testrun.proto

```diff
@@ -0,0 +1,257 @@
+// Copyright 2014 Google Inc. All Rights Reserved.
```

Comment:
Oh I didn't even notice that. I meant that the file itself is actually out of date, since it's missing the MULTIDIM_JSON and related changes